### DISCUSSION TITLE
Insulin Pens - Changing insulin types

### DISCUSSION THREAD

**Discussion by Loquacity** [NightscoutFoundation/xDrip#3322] [Q&A]
Hi everyone,

I'm using two 'smart' insulin pens, both are NovoPen 6, and both are currently being used with my basal insulin (Levemir, although they're saved in xDrip as Lantus, since Levemir was not an available insulin type at the time). With the release of my bolus insulin (Fiasp) in a penfill, I am hoping to switch to using one smart pen for bolus instead, and the other for basal. I thought that if I could get xDrip to 'forget' my pens, then I could re-add them on the next dose, and be able to set them up with the correct insulin, but I can't see a way to do that. Is there some way to either have xDrip 'forget' the pen, or a way to manually change which type of insulin the pen is using? Thanks for all the help!

**Comment by Navid200** [comment]
@jamorham

**Comment by Navid200** [comment]
What happens if you use the Android Bluetooth settings on the phone to forget the device?
Then, attempt to use the pen as a new device with the correct insulin type.  Will it not let you do that?

**Reply by Loquacity** [reply]
Erm, they're not Bluetooth, they're NFC.

**Reply by Navid200** [reply]
Have you seen this open issue?
https://github.com/NightscoutFoundation/xDrip/issues/2074

You can post in there and ask as well.

**Reply by Loquacity** [reply]
Somehow, I had completely missed that issue in my searches! Thanks, I've commented there.
